pom.xml

```diff
@@ -1,298 +1,244 @@
-     <version>2.9</version>
-    </plugin>
-    <plugin>
-     <artifactId>maven-release-plugin</artifactId>
-     <version>2.2.1</version>
-    </plugin>
-    <plugin>
-     <artifactId>maven-resources-plugin</artifactId>
-     <version>2.5</version>
-    </plugin>
-    <plugin>
-     <artifactId>maven-site-plugin</artifactId>
-     <version>2.3</version>
-    </plugin>
-    <plugin>
-     <artifactId>maven-source-plugin</artifactId>
-     <version>2.1.2</version>
-    </plugin>
-    <plugin>
-     <artifactId>maven-surefire-plugin</artifactId>
-     <version>2.9</version>
-     <configuration>
-      <forkMode>always</forkMode>  <!-- because H2database is first shutdown when JVM exit -->
-     </configuration>
-    </plugin>
-    <plugin>
-     <artifactId>maven-war-plugin</artifactId>
-     <version>2.1.1</version>
-    </plugin>
-   </plugins>
-  </pluginManagement>
- </build>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>dk.statsbiblioteket.licensemodule</groupId>
+	<artifactId>licensemodule</artifactId>
+	<version>4.0-SNAPSHOT</version>
+	<name>licensemodule</name>
+	<packaging>war</packaging>
+
+
+	<parent>
+		<groupId>org.sbforge</groupId>
+		<artifactId>sbforge-parent</artifactId>
+		<version>22</version>
+	</parent>
+
+	<properties>
+		<build.time>${maven.build.timestamp}</build.time>
+		<maven.build.timestamp.format>yyyy-MM-dd HH:mm</maven.build.timestamp.format>
+		<!-- Specify java version here, to support forbiddenapis plugin -->
+		<maven.compiler.source>1.8</maven.compiler.source>
+		<maven.compiler.target>1.8</maven.compiler.target>
+		<!-- Enable forbiddenapis plugin, do disable set to none -->
+		<api.check.phase>process-test-classes</api.check.phase>
+	</properties>
+
+	<repositories>
+		<repository>
+			<id>sbprojects-nexus</id>
+			<url>https://sbprojects.statsbiblioteket.dk/nexus/content/groups/public</url>
+			<releases>
+				<enabled>true</enabled>
+			</releases>
+			<snapshots>
+				<enabled>true</enabled>
+			</snapshots>
+		</repository>
+	</repositories>
+
+
+
+	<dependencies>
+
+		<dependency>
+			<groupId>dk.statsbiblioteket.sbutil</groupId>
+			<artifactId>sbutil</artifactId>
+			<version>0.5.56</version>
+		</dependency>
+
+		<!-- DB -->
+		<dependency>
+			<groupId>com.h2database</groupId>
+			<artifactId>h2</artifactId>
+			<version>1.3.170</version>
+			<scope>test</scope>
+		</dependency>
+
+		<dependency>
+			<groupId>org.postgresql</groupId>
+			<artifactId>postgresql</artifactId>
+			<version>9.4.1208</version>
+		</dependency>
+
+
+		<dependency>
+			<groupId>org.apache.commons</groupId>
+			<artifactId>commons-dbcp2</artifactId>
+			<version>2.0</version>
+		</dependency>
+
+
+		<dependency>
+			<groupId>junit</groupId>
+			<artifactId>junit</artifactId>
+			<version>4.8.2</version>
```

Comment:
Please update junit framework to junit5 (jupiter)

Reply:
Done. Minor rewrite of test-code